api/capi/src/nnstreamer-capi-single.c

```diff
@@ -559,7 +561,11 @@ ml_single_invoke (ml_single_h single,
   if (!sample) {
     ml_loge ("Failed to get the result from sink element.");
     single_h->clear_previous_buffer = TRUE;
+#ifdef ML_SINGLE_SUPPORT_TIMEOUT
```

Comment:
inside the feature (appsink without timeout feature does not need to clear old buffer)

```
#ifdef ML_SINGLE_SUPPORT_TIMEOUT
    single_h->clear_previous_buffer = TRUE;
```

Reply:
Right, updated.